CODEOWNERS

```diff
@@ -31,9 +31,9 @@ extensions/filters/common/original_src @snowp @klarose
 # sni_cluster extension
 /*/extensions/filters/network/sni_cluster @rshriram @lizan
 # tracers.datadog extension
-/*/extensions/tracers/datadog @cgilmour @palazzem
+/*/extensions/tracers/datadog @cgilmour @palazzem @mattklein123
 # tracers.xray extension
-/*/extensions/tracers/xray @marcomagdy @lavignes
+/*/extensions/tracers/xray @marcomagdy @lavignes @htuch
```

Comment:
I think it would be better to have someone with stronger tracing knowledge be the maintainer owner here.

Reply:
Have any suggestions? You merged the PR so I'm inclined to let you sort out who the owner should be :-)

Reply:
I think @mattklein123 is the only maintainer with deep knowledge of tracing. This is probably a serious gap. @envoyproxy/maintainers any volunteers who want to build deeper tracing knowledge?

Reply:
Actually I'd pinged #9167 about this suggesting jmarantz since he was the other maintainer who reviewed and owns fewer things.  Josh WDYT?